config/sac/Pyramids.yaml

```diff
@@ -5,32 +5,32 @@ behaviors:
       learning_rate: 0.0003
       learning_rate_schedule: constant
       batch_size: 128
-      buffer_size: 500000
-      buffer_init_steps: 10000
+      buffer_size: 5000000
+      buffer_init_steps: 1000
       tau: 0.01
       steps_per_update: 10.0
       save_replay_buffer: false
       init_entcoef: 0.01
       reward_signal_steps_per_update: 10.0
     network_settings:
       normalize: false
-      hidden_units: 256
-      num_layers: 2
+      hidden_units: 512
```

Comment:
Did you run with just the larger buffer?

Reply:
Yes, it does get a better reward but it peaks out much earlier (around 1.2-ish) and the losses continue to climb.